apps/api/src/app/workflows-v2/util/template-parser/liquid-parser.ts

```diff
@@ -162,7 +165,11 @@ function extractProps(template: any): { valid: boolean; props: string[]; error?:
    * Invalid: {{user.first name}} - postfix length would be 2 due to space
    */
   if (initial.postfix.length > 1) {
-    return { valid: false, props: [], error: 'Variables with spaces are not supported' };
+    return {
+      valid: false,
+      props: [],
+      error: `<span>Invalid variable name containing whitespaces. Variables must follow the ${DOT_ANNOTATION_MARKDOWN_HREF}</span>`,
```

Comment:
I'm unsure about this approach to building HTML code on the API side. If this API is used by RN or any other platform different from the web, the error message would be useless. 
Instead, we should return the pure message to the Dashboard and have a hint or a link in the UI pointing to our documentation teaching how to use the editors.
cc @SokratisVidros

Reply:
The assumption right now is that the only consumer of the new API is the Dashboard, this is why this approach felt appropriate to me.

Reply:
I agree with Pawel. We also use react, so returning HTML from a JSON API will eventually bit us. 

See also this that is strictly relevant. The complexity for error rendering should be in the Dashboard UI. The message text is a the responsibility of the API.

Reply:
Ok so it is decided I will remove the link, but please take into consideration that providing links will become much more complex in the client.